automationtestapp/src/main/java/com/microsoft/aad/automation/testapp/ResultActivity.java

```diff
@@ -61,6 +66,10 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         mTextView.setText(resultText);
 
+        mLogView = (TextView) findViewById(R.id.adalLogs);
+        mLogView.setMovementMethod(new ScrollingMovementMethod());
```

Comment:
It depends on the requirement. We don't need to set the movement method for the text box if we don't need to show all the logs.